### PR TITLE
Do not filter on linux/amd64 per default

### PR DESCRIPTION
**Pull request by s00500**
Tasks with no platform would otherwise not perform on runners
with different OS/ARCH combos than linux/amd64

Signed-off-by: Lukas Bachschwell <lukas@lbsfilm.at>

### CODE REVIEW

File: server/grpc/rpc.go [hunk 1]
```diff
@@ -459,6 +459,10 @@ func createFilterFunc(filter rpc.Filter) (queue.Filter, error) {
 		}
 
 		for k, v := range filter.Labels {
+			if k == "platform" && task.Labels[k] == "" {
```
[the remark this answers is not in the document]
Comment (s00500) [reply server/grpc/rpc.go:462]: Fact is that skipping the empty platform label definitely changes the current strange behavior to what one would expect

**Comment by 6543** [comment]
@s00500 it would be nice if you could add a unit test for the filter.go .... - so we can test and preserver the new behaviour

if you need help with doing so just tell me on discord

**Comment by 6543** [comment]
... done

**Review by 6543 — APPROVED** [review]

**Comment by s00500** [comment]
Hihi would definitely have done it, but tomorrow, coding at a bar is not so easy ;-) Thanks a lot!

**Pull request merged**